ts/utils/utils.ts

```diff
@@ -556,4 +556,17 @@ export const utils = {
 
         return `0x${_n.toString(16).padStart(64, '0')}`;
     },
+    getPoolDisplayName({ poolId, metaData }: { poolId: string; metaData?: { name?: string } }): string {
```

Comment:
nit: (no preference either way) would this better belong in staking_utils instead?